selfdrive/test/process_replay/test_processes.py

```diff
@@ -42,7 +42,7 @@
 segments = [
   ("BODY", "regenA67A128BCD8|2024-08-30--02-36-22--0"),
   ("HYUNDAI", "regen9CBD921E93E|2024-08-30--02-38-51--0"),
-  ("HYUNDAI2", "regen12E0C4EA1A7|2024-08-30--02-42-40--0"),
+  ("HYUNDAI2", "regen306779F6870|2024-10-03--04-03-23--0"),
```

Comment:
the Hyundai interaction checker was blocking enable as the original route didn't log button events if stock long. now it logs buttons all the time